src/rust.rs

```diff
@@ -68,7 +68,7 @@ use panic::maybe_resume_unwind;
 
 use default_heapsize;
 
-pub use mozjs_sys::jsgc::GCMethods;
+pub use mozjs_sys::jsgc::{GCMethods, IntoHandle};
```

Comment:
We `use IntoHandle as IntoRawHandle` at line 12 (https://github.com/servo/rust-mozjs/pull/445/files#diff-34b82cd514c7429b32451dd0b07c1e1bR12).

Not sure I should `pub` there or re-export here?
But I think `pub` at line 12 will make the re-export trait name to be `IntoRawHandle` ?